examples/end-to-end-examples/support_chatbot/README.md

```diff
@@ -0,0 +1,179 @@
+
+## Step 1: Getting Our Data
+
+Let's first start with downloading the github repository that we want to finetune our model on so that it can get a basic understanding of the codebase. The [repo_downloader](./scripts/repo_downloader.py) will clone the git repository into a designated cloning directory where it will flatten the repository to be an assortment of text files in a local directory: `retrieval_data/{REPOSITORY_NAME}`. It will then erase the cloning directory.
+
+```bash
+python scripts/repo_downloader.py REPO_LINKS
+```
+
+**Fields to replace with your values:** `REPO_LINKS` (in the command line). For instance, to download the MosaicML Composer repository, we run:
+```bash
+python scripts/repo_downloader.py https://github.com/mosaicml/composer
+```
+
+## Step 2: Converting to MDS
+
+As mentioned in the [MosaicML platform setup](#mosaicml-platform-setup) section, the MosaicML platform does not have permanent storage on the compute nodes. This means that all data will be streamed in and out from a cloud provider. In order to make this process as efficient as possible during a training run, we will convert the data into a format that is optimized for streaming, using our [Streaming](https://github.com/mosaicml/streaming) library. This format is called [MDS](https://docs.mosaicml.com/projects/streaming/en/stable/fundamentals/dataset_format.html#mds), and is a simple format that is optimized for streaming.
+
+This example will contain a script for a [text to MDS](./scripts/conversion/convert_txt_to_stream.py) conversion. For text to MDS conversion, we will run:
+
+```bash
+mcli run -f mcli_yamls/conversion/convert_txt_to_stream.yaml --cluster CLUSTER
+```
+
+**The conversion YAMLs will not work on local directories if the data is not git pushed onto the repository linked by the YAML.** This means that it will not be able to recognize and oci path to data, but only a local path in the github repository (you can feed it the relative path from *step 1*). Thus, if you choose to use the YAML method, make sure that you push the data downloaded from *step* one to your repository. That being said, github repositories are typically very small and you can probably just run this locally:
```

Comment:
```suggestion
**The conversion YAMLs will not work on local directories if the data is not git pushed onto the repository linked by the YAML.** This means that it will not be able to recognize a remote path to data, but only a local path in the github repository (you can feed it the relative path from *step 1*). Thus, if you choose to use the YAML method, make sure that you push the data downloaded from *step* one to your repository. That being said, github repositories are typically very small and you can probably just run this locally:
```